### DISCUSSION TITLE
models/276/CeO2-ZrO2-SCAN+rVV10-model

### DISCUSSION THREAD

**Discussion by giscus** [deepmodeling/AIS-Square#236] [Announcements]
# models/276/CeO2-ZrO2-SCAN+rVV10-model

https://www.aissquare.com/models/detail?pageType=models&name=CeO2-ZrO2-SCAN%252BrVV10-model&id=276

**Comment by QuantumMisaka** [comment]
SCAN+rVV10使用哪个软件，通过何种基组计算出来的呢？
Which software and basis set do you use for SCAN+rVV10 functional ?

**Reply by ZHANG-JINYU-1994** [reply]
你好，我们采用的VASP进行的相关计算，设置可以参考https://www.vasp.at/wiki/index.php/Nonlocal_vdW-DF_functionals。具体的INCAR文件会在文章发表后随补充材料给出。

**Reply by ZHANG-JINYU-1994** [reply]
INCAR请见引文[1]的补充材料

**Comment by ZHANG-JINYU-1994** [comment]
你好，我们采用的VASP进行的相关计算，设置可以参考https://www.vasp.at/wiki/index.php/Nonlocal_vdW-DF_functionals。具体的INCAR文件会在文章发表后随补充材料给出。

**Comment by suzhed** [comment]
您好，请问一下准备lammps的data文件，type_map的原子顺序是什么样的，是 Ce-1，Zr-2，O-3 这样的么？

**Reply by ZHANG-JINYU-1994** [reply]
此页面介绍部分已提到：Type 1, 2, and 3 are O, Zr, and Ce respectively.
